setup.cfg

```diff
@@ -1,5 +1,4 @@
 [tool:pytest]
-addopts = -nauto -s
```

Comment:
why do you want to remove this? Doesn't this parallelize our tests?

Reply:
Well it was explicitly disabled on Travis (in another file), and it seems this one was never used because the lack of a space between `-n` and `auto` would break things. So I think this has only ever existed in our collective imagination -- may as well remove it.

Reply:
`-nauto` vs `-n auto` certainly doesn't break things, unless pytest changed something.

it was disabled because Travis machines had 2 cores, but parallelizing actually slowed the tests down. You might want to try it here to see if it makes your tests faster.

Reply:
I think lets just avoid it for now. IIRC we had issue with the memory usage tests acting funny when we ran the test suite in parallel.

Reply:
Yeah we still only get 2 cores: [see here](https://docs.github.com/en/actions/reference/specifications-for-github-hosted-runners)